### PR TITLE
test full pipeline

### PR DESCRIPTION
**Pull request by jakebiesinger**
I've added 1K test reads from hg19 chromosome 18 (`chr18[12000:12000+1000]`).

It seems we've been having some problems with travis not failing where it should.  This will at least start an NC and CC, though it's not using HDFS at all.

### CODE REVIEW

**Comment by jakebiesinger** [comment]
Any thoughts, @Nan-Zhang @JavierJia @anbangx ?

File: .travis.yml [hunk 1]
```diff
@@ -16,3 +16,6 @@ script:
     - cd genomix
     - mvn package -q -e -Djava.util.logging.config.file=genomix-driver/src/main/resources/conf/logging.properties
     - ls -t */target/surefire-reports/*.txt | xargs tail -n +1  # show test reports in the order they ran
+    - cd genomix-driver/target/genomix-driver-0.2.10-SNAPSHOT-binary-assembly
```
Comment (JavierJia) [inline .travis.yml:22]: Is this directory name correctly ? `genomix-driver-0.2.10-SNAPSHOT-binary-assembly` . 
What's the different with this one and `appassembler` ?

**Comment by JavierJia** [comment]
The last message showed in Travis is below

```
The authenticity of host 'localhost (::1)' can't be established.
ECDSA key fingerprint is 04:bb:92:85:dc:2b:78:3d:c0:a1:85:7e:b3:1f:52:89.
Are you sure you want to continue connecting (yes/no)? 
No output has been received in the last 10 minutes, this potentially indicates a stalled build or something wrong with the build itself.
The build has been terminated
```

If we run the startXX in some new machine, I think we should make the ssh authentication first.

File: .travis.yml (hunk 1 above)
Comment (jakebiesinger) [inline .travis.yml:22]: Glad you asked.  Turns out `appassembler` is a temporary stop on the way to this `binary-assembly` directory.  The difference is that the binary assembly has been `chmod`'ed properly.  You'll also find `.zip` and `.tar.gz` binary assemblies.  If we want to distribute our project, we'd do so by giving our those zipped assemblies.

In other words, you should be running from this directory, _not_ from appassembler.

Comment (JavierJia) [inline .travis.yml:22]: Got it.

**Comment by JavierJia** [comment]
OK, but that question seemed need to `type` yes ? 

```
Are you sure you want to continue connecting (yes/no)? 
```

**Comment by jakebiesinger** [comment]
Yeah, I'm working on that now :)

**Comment by jakebiesinger** [comment]
Took a bit of doing, but this should work. The Travis log shows it's not hanging anymore.

**Comment by anbangx** [comment]
Great. Look good to me!

**Comment by JavierJia** [comment]
Passed the test. :+1: 
LGTM

**Comment by jakebiesinger** [comment]
```
Oct 23, 2013 4:47:34 AM edu.uci.ics.genomix.driver.GenomixDriver runGenomix
INFO: Finished the Genomix Assembler Pipeline in 65685ms!
The command "bin/genomix -kmerLength 55 -pipelineOrder BUILD_HYRACKS,MERGE,TIP_REMOVE,MERGE,BUBBLE,MERGE -localInput ../../src/test/resources/data/sequence/pathmerge/hg19.chr18.skip12K.first1K.1Kreads/" exited with 0.
```

Woohoo! Travis is doing genome assembly!!!

**Comment by jakebiesinger** [comment]
Okay, @Nan-Zhang you seem to be slow to respond to these... I guess I'll go ahead and merge without you...

**Comment by Nan-Zhang** [comment]
sorry, @jakebiesinger  I thought l am behind of this stuff, after completing adding the genomix-data tests, I will run some real data for tests

**Pull request merged**